cli/src/pcluster/validators/fsx_validators.py

```diff
@@ -48,6 +48,28 @@ def _validate(
             )
 
 
+class FsxDraValidator(Validator):
+    """
+    FSX Dra validator.
+
+    Verify compatibility of given S3 association options for FSX.
+    """
+
+    def _validate(self, data_repository_associations, import_path, export_path):
+        if data_repository_associations and (import_path or export_path):
+            self._add_failure(
+                "When specifying data repository associations, import and export path "
+                "can not be used on the same file system.",
+                FailureLevel.ERROR,
+            )
```

Comment:
Is this validation already done in `validate_s3_association_unsupported_parameters`?